### PR TITLE
Update addFlight methods in android and ios

### PR DESCRIPTION
**Pull request by humphriest**

### CODE REVIEW

File: Android.md [hunk 1]
```diff
@@ -118,10 +118,14 @@ The deeplink type `cross-sell` is used for shortening a users search by pre-popu
...
 ```java
 //  Add this call after `MobilitySDKManager.init` and before `MobilitySDKManager.showMobility`
-MobilitySDKManager.addFlight("<origin-IATA>", "<destination-IATA>", "<flight-number>", "<flight-date-time>");
+MobilitySDKManager.addFlight("<origin-IATA>", "<destination-IATA>", "<flight-number>", "<flight-departure-date-time>", "<flight-arrival-date-time>", "<origin-airport-name>", "<destination-airport-name>","<direction>");
 ```
 
-All of the paramaters are strings. The `<flight-date-time>` is in the format YYYY-MM-DDThh:mm:ss.s.
+All of the parameters are strings except for `<direction>`.
+
+`<direction>` is an **enum** with the values of either `DeeplinkDirection.DROPOFF_TO_ORIGIN` or `DeeplinkDirection.PICKUP_FROM_DESTINATION`. `DeeplinkDirection.DROPOFF_TO_ORIGIN` is used when the user is to be dropped off at the airport they are flying out of from the flight details provided. `DeeplinkDirection.PICKUP_FROM_DESTINATION` is used when the user is to be picked up at the airport they are landing at from the flight details provided.
```
Comment (gianpaj) [inline Android.md]:
```suggestion
`<direction>` is an **enum** with the values of either `DeeplinkDirection.DROPOFF_TO_ORIGIN` or `DeeplinkDirection.PICKUP_FROM_DESTINATION`.
`DeeplinkDirection.DROPOFF_TO_ORIGIN` is used when the user on the transport (taxi, etc.) is to be dropped off at the airport they are flying out of, using the flight details provided.
`DeeplinkDirection.PICKUP_FROM_DESTINATION` is used when the user on the transport (taxi, etc.) is to be picked up at the airport they are landing at, using the flight details provided.
```

or 

`DeeplinkDirection.DROPOFF_TO_ORIGIN` should be specified if the user should be dropped off at the airport the flight is flying from – using the flight details provided.

`DeeplinkDirection.PICKUP_FROM_DESTINATION` should be specified if the user should be picked up at the airport the flight is flying to – using the flight details provided.

Same wording for iOS

**Review by gianpaj — CHANGES_REQUESTED** [review]
Just a wording suggestions on the `direction` param

File: Android.md (hunk 1 above)
Comment (humphriest) [reply Android.md]: I don't like the wording of this ` when the user on the transport (taxi, etc.)` but I see what you mean.
Maybe something like:
`DeeplinkDirection.DROPOFF_TO_ORIGIN` is used when the user needs a ride to the airport they are flying out of, using the flight details provided.
or
`DeeplinkDirection.DROPOFF_TO_ORIGIN` is used when the user is to be dropped off, by taxi or other, at the airport they are flying out of, using the flight details provided.

Comment (gianpaj) [reply Android.md]: ok, yes. those look good to me

Comment (humphriest) [reply Android.md]: Updated

**Pull request merged**